scss/_config.scss

```diff
@@ -0,0 +1,15 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+$primary-color: #0699A6;
```

Comment:
Nice thoughts declaring all your colors and font sizes before hand. Since we already have a design, maybe you can take this a little further. 

Declare all font sizes, font weights, colours and even font families used. I recommend having a separate file for variable declarations. If there are too many variables, you can further split them into different files depending on their types.